### ISSUE TITLE
Make SELENIUM_OVERRIDES available to override from base package.json

### ISSUE THREAD

**Issue by philippw** [Polymer/wct-local#75]
I'm trying to convince wct-local to download selenium and its drivers via an internal Artifactory server instead of directly from GitHub. selenium-standalone supports this via its baseURL settings.

However, as selenium-standalone is a transitive dependency of wct-local I cannot override the default values.
Fortunately, there's the scripts/postinstall.js and its SELENIUM_OVERRIDES, which would allow for excatly that, would it not be the case that it only ever parses settings fromwct-local's ownpackage.json`.

Can we change the lookup path from ../package.json to ../../../package.json to be able to override the Selenium settings from the base package.json?

Bug migrated from the [web-component-tester issues](https://github.com/Polymer/web-component-tester/issues/675).

**Comment by philippw** [comment]
Maybe we can even expose it as an environment variable?
Quoting @nicojs:

> I think using an environment variable for this is more common for these types of features. Also, this
> issue is reported to the wrong git repository. I think it should be reported to wct-local
>
> See this file: https://github.com/Polymer/wct-local/blob/master/scripts/postinstall.js

**Comment by bernardoVale** [comment]
@philippw you mean an env variable with package.json path?

**Comment by philippw** [comment]
@bernardoVale Yes and no. I'd like to be able to overwrite it with config values from your base `package.json` like this:

```
"selenium-overrides": {
    "version": "<version>",
    "baseURL": "URL-to-your-cache",
    "chrome": {
      "version": "<version>",
      "arch": "process.arch",
      "baseURL": "URL-to-your-cache"
    },
    "firefox": {
      "version": "<version>",
      "arch": "process.arch",
      "baseURL": "URL-to-your-cache"
    }
  }
```

If you just overwrite the `SELENIUM_OVERRIDES`, it'd have to be a relative path to your base config.

**Comment by bernardoVale** [comment]
take a look at my PR, I've tested locally and it works fine.

```
SELENIUM_OVERRIDES_CONFIG='../selenium-config.json' npm install
```

but it could be your `package.json`. It will look for a `selenium-overrides` object inside any json file provided in this env variable
